### PR TITLE
Add Container cleanup Github Action

### PR DESCRIPTION
**Pull request by laurensWe**
# Description

Describe in detail the changes you are proposing, and the rationale.

Link all GitHub issues fixed by this PR.
Make sure to first open an issue, get community approval and only then create Pull Request to resolve it.
All Pull Requests must have an issue attached to them

Resolves #

## Checklist

Please check all the boxes that apply to this pull request using "x":

- [ ]  I have tested the changes locally and verified that they work as expected.
- [ ]  I have followed the project's coding conventions and style guidelines.
- [ ]  I have rebased my branch onto the latest commit of the main branch.
- [ ]  I have squashed or reorganized my commits into logical units.
- [ ]  I have read, understood and agree to the [Developer Certificate of Origin](../blob/main/DCO.md), which this project utilizes.

### CODE REVIEW

**Review by berrydenhartog — APPROVED** [review]
LGTM

File: .github/workflows/ghcr-cleaner.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      - name: Delete untagged images
+        uses: dataaxiom/ghcr-cleanup-action@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: 'Clean up development images'
+        uses: dataaxiom/ghcr-cleanup-action@v1
+        with:
+          keep-n-tagged: 10
+          exclude-tags: '*release*,*main*,*latest*'
```
Comment (berrydenhartog) [inline .github/workflows/ghcr-cleaner.yml:19]: een release tag bestaat vaak uit v1.2.3 bijvoorbeeld. denk dat deze nu verwijderd word

File: .github/workflows/ghcr-cleaner.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+name: Github Container Registry Cleaner
+on:
+  schedule:
+    - cron: "0 4 * * Sun"
+
+jobs:
+  delete_old_images:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Delete untagged images
```
Comment (berrydenhartog) [inline .github/workflows/ghcr-cleaner.yml:10]: Does this not delete everything?

File: .github/workflows/ghcr-cleaner.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+          keep-n-tagged: 10
+          exclude-tags: '*release*,*main*,*latest*'
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: 'Clean up images from main'
+        uses: dataaxiom/ghcr-cleanup-action@v1
+        with:
+          keep-n-tagged: 5
+          tags: '*main*'
+          token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (uittenbroekrobbert) [inline .github/workflows/ghcr-cleaner.yml:27]: I checked the repository and there is only one main tag. So I don't think this job would do anything. @berrydenhartog is that correct?

File: .github/workflows/ghcr-cleaner.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
+name: Github Container Registry Cleaner
+on:
+  schedule:
+    - cron: "0 4 * * Sun"
```
Comment (uittenbroekrobbert) [inline .github/workflows/ghcr-cleaner.yml:4]: We could consider running this cleaner daily. We generate quite some images.

**Review by uittenbroekrobbert — APPROVED** [review]
LGTM!

**Review by berrydenhartog — APPROVED** [review]
LGTM

**Pull request merged**